lua/lspsaga/signaturehelp.lua

```diff
@@ -16,7 +16,7 @@ local function check_server_support_signaturehelp()
   end
   local clients = vim.lsp.buf_get_clients()
   for _, client in pairs(clients) do
-    if client.resolved_capabilities.signature_help == true then
+    if client.server_capabilities.signatureHelpProvider == true then
```

Comment:
Should I use `~= nil` as condition? For readability.

Reply:
Seems like somebody is already doing great job in #106 , maybe I should close this PR.

Reply:
> Should I use `~= nil` as condition? For readability.

I'm not so sure about this one, I don't know if there's no provider it will be nil, are you sure about this? 
if not, we could avoid that with this

```lua
if client.server_capabilities.signatureHelpProvider then
```